scout/server/blueprints/variants/templates/variants/variants.html

```diff
@@ -130,17 +130,23 @@ <h4 class="panel-title">
   </a>
   {% set comment_count = variant.comments.count() %}
   {% if variant.acmg_classification %}
-    <span class="badge pull-right" title="{{ variant.acmg_classification.label }}">
+    <span class="badge pull-right" title="{{ variant.acmg_classification.label }}">
       {{ variant.acmg_classification.short }}
     </span>
   {% elif variant.manual_rank %}
     <span class="badge pull-right">{{ variant.manual_rank }}</span>
   {% elif comment_count > 0 %}
-    <!-- At least show if there are any comments written -->
-    <span class="badge pull-right">
+    <a href="#" 
+       class="badge pull-right" 
+       data-toggle="popover" 
+       data-placement="right" 
+       data-html="true" 
+       data-trigger="hover click" 
+       data-content="{{ comments_table(institute, case, variant.comments, variant._id) }}"
```

Comment:
hehe I didn't even know we loaded comments when we displayed the variants list 😅 -- kind of pointless unless we show them.........